<ui version="4.0" >
 <class>ConfigureMIDIOut</class>
 <widget class="QDialog" name="ConfigureMIDIOut" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configure MIDI Output Plugin</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="4" >
    <widget class="QTreeWidget" name="m_list" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>Output</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>MIDI Channel</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Mode</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QPushButton" name="m_refreshButton" >
     <property name="text" >
      <string>Refresh</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="m_editButton" >
     <property name="text" >
      <string>Edit...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>121</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureMIDIOut</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>317</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel" >
     <x>200</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureMIDIOut</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>384</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel" >
     <x>238</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
